macros/fivetran_lookback.sql

```diff
@@ -0,0 +1,18 @@
+{% macro fivetran_lookback(from_date, datepart, interval, safety_date='2010-01-01') %}
+
+{{ adapter.dispatch('fivetran_lookback', 'fivetran_utils') (from_date, datepart, interval, safety_date='2010-01-01') }}
+
+{%- endmacro %}
+
+{% macro default__fivetran_lookback(from_date, datepart, interval, safety_date='2010-01-01')  %}
+
+    {% set sql_statement %}
+        select coalesce({{ from_date }}, {{ "'" ~ safety_date ~ "'" }})
+        from {{ this }}
+    {%- endset -%}
+
+    {%- set result = dbt_utils.get_single_value(sql_statement) %}
```

Comment:
I actually don't have a recollection of using the `dbt_utils.get_single_value()` macro before. Is this essentially doing the same thing as this [run_query](https://github.com/fivetran/dbt_salesforce/blob/fa353308688ed708ea2da5f8116f21ec8aba4989/models/salesforce/intermediate/int_salesforce__date_spine.sql#L13) operation? If so, the get_sing_value is much cleaner and probably what we should be using across our models!

Reply:
It is slightly different execution, but the end result looks like it's practically the same. [Here](https://github.com/dbt-labs/dbt-utils/blob/main/macros/sql/get_single_value.sql) is the source for `get_single_value()`. I found it by accident while reading documentation and thought I'd give it a try to make things easier.